package.json

```diff
@@ -41,7 +41,7 @@
     "safe-json-parse": "4.0.0",
     "static-config": "2.1.0",
     "tape-cluster": "2.0.1",
-    "tchannel": "3.2.3",
+    "tchannel": "3.3.1",
```

Comment:
Why even did a test change upgrade tchannel in passing? Please don't just slip in tchannel upgrades without explicit approval, especially after initial approval like happened on this PR.